client.go

```diff
@@ -1018,12 +1018,17 @@ func (client *gocloak) GetGroups(token string, realm string, params GetGroupsPar
 }
 
 // GetGroupsCount gets the groups count in the realm
-func (client *gocloak) GetGroupsCount(token string, realm string) (int, error) {
+func (client *gocloak) GetGroupsCount(token string, realm string, params GetGroupsParams) (int, error) {
```

Comment:
It would be a non breaking change if we use "params ...GetGroupsParams" any thoughts on this? 
Do you think that would be uglier code doing it that way?